task9/view.js

```diff
@@ -0,0 +1,205 @@
+'use strict';
+
+import {testContainer} from "./postcollection.js";
```

Comment:
Trying to start it locally I get such error. Let's discuss it during lesson.
![image](https://user-images.githubusercontent.com/25108879/82849815-513e2680-9f02-11ea-9934-9362ab934963.png)